src/VirtualClient/VirtualClient.Contracts/Parser/DataTableExtensions.cs

```diff
@@ -138,7 +138,7 @@ public static DataTable ConvertToDataTable(string text, IList<KeyValuePair<int,
         /// <param name="tags">Tags</param>
         /// <param name="ignoreFormatError">If this method ignore FormatException. Default is false.</param>
         /// <returns>Parsed Metrics from a data table.</returns>
-        public static IList<Metric> GetMetric(
+        public static IList<Metric> GetMetrics(
```

Comment:
Just a reminder to us both, to update internal extension repo on this package

Reply:
Yep. I will be following up this PR merger with the updates to our ADO repo.